projects/managed-hubs/support.md

```diff
@@ -173,24 +174,73 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
-   2. If you don't believe you can resolve the issue in 30 minutes, jump to the next step.
-3. **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
-   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. Add the {guilabel}`support` label to it.
-4. **Communicate status**. Once we have an issue created to track the next steps, send a message to the Community Representative letting them know about the situation and what the next steps will be. Here's a template to help guide you:
+The goal of the non-incident response process is to bring standardization to our support response. This simple workflow tries to battle the bias towards a reactive response whereas it is also bringing some common patterns so all of our non-incident support responses are cohesive and shared among our support stewards.
+The current iteration of the workflow states each step and who should be responsible/accountable for the specific step, plus some other clarifications.
+
+When a new ticket lands in Freshdesk under the support group and it is not an incident, you should follow the following steps:
+
+1. `Who: Support steward`
+
+   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know a time-boxed investigation will start soon. Please request any additional information you may need to be able to reproduce the issue in step 2.
+
+2. `Who: Support steward`
+
+   **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
+   1. If you resolve the issue, then jump to the "Confirm resolution" step 7.
+   2. If you don't believe you can resolve the issue (or you couldn't) in 30 minutes, jump to the next step.
+
+3. `Who: Support Steward`
+
+   **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and/or you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
+
+   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. It should be automatically added to the **Eng & Prod** board by the existing automation. Add the **type**: `support` and choose an **impact** level in the **Eng & Prod** board metadata (available for the very same issue, no need to visit the board itself!).
+   
+   If the issue has a `critical` impact (we defer that first evaluation to the support steward), an additional ping to the support Slack channel is needed to boost the signal.
```

Comment:
@2i2c-org/engineering, what do people think about the above list for "critical" support issues?
Am I missing something? Am I adding too many items? Thoughts?

Reply:
Not sure if I think access to specific buckets should be considered critical, and I could see various authentication and authorization updates as critical and non-critical depending on their nature.

"Image updates" are a bit vague in my mind since I'm thinking we are not responsible for the user environment images - unless we are in certain exceptions. Is our agreements in those exceptions that we should help quickly as well?

Reply:
I think access to buckets, auth and authz can be considered critical depending on their nature. So, I should probably clarify that further.
With "image updates" I am only trying to refer to the update on the reference to an image, such as https://github.com/2i2c-org/infrastructure/pull/3034/files, I should probably be more specific about it!
Thanks for the feedback!! I will update the list accordingly.

Reply:
@consideRatio, wdyt about this one?
```
Examples of those sorts of issues (requests) are:
     * Image refs updates
     * Profile updates
     * User storage limitations
     * Grafana (and Prometheus) failures
Additionally and depending on the nature AND context of the issue (request):
     * Access to specific buckets
     * Authentication and authorization updates
```

Reply:
Looks great @damianavila!!

Reply:
Updating the suggestion as per the previous message and committing now.